lib/Fhp/Dialog/Dialog.php

```diff
@@ -174,9 +174,7 @@ public function sendMessage(AbstractMessage $message, $tanMechanism = null, \Clo
 				throw $ex;
 			}
 
-			$hitan = $response->splitSegment($response->findSegment('HITAN'));
-
-			if (!isset($hitan[4]) or $hitan[4] == 'nochallenge') {
```

Comment:
Instead of looking for HITAN fields, `$response->isStrongAuthRequired()` checks for a message with code 3076, which, according to the specs, indicates that no auth is required.

Reply:
:heavy_check_mark: